math/gnuplot/Portfile

```diff
@@ -129,7 +129,16 @@ variant old_bitmap_terminals description "Enable PBM (Portable Bit Map) and othe
     configure.args-replace      --without-bitmap-terminals --with-bitmap-terminals
 }
 
-default_variants                +aquaterm +luaterm +pangocairo +wxwidgets +x11
+default_variants                +luaterm +pangocairo +x11
+
+# aquaterm is currently not building on 10.4
```

Comment:
This PR looks ok.

Nonetheless I would be curious in what exactly is failing on Tiger. I thought that AquaTerm would build on Tiger, but I don't remember all the details any longer (maybe the code itself is compatible, but the project doesn't build with an older xcode).

Reply:
Ken's TigerPorts had pegged an older aquaterm; I tried fussing with the newer version but didn't make much progress. I can open a Trac ticket about it once I re-run the build and have a proper log. IIRC the build was using the system GCC contrary to the MP compiler settings, but I can't tell you how far the rabbit hole goes...

Reply:
AquaTerm hasn't been touched for ages. The final few commits just added support for 64-bit 10.6, and maybe those few commits unintentionally broke support for 10.4.

If it's about using the right compiler, it's a different issue.

Please do open a track ticket when you come around to testing it.

Reply:
What was the ultimate outcome of this? Does a new issue need to be created, or... ?

Reply:
@mascguy Stock MP aquaterm does not compile on Tiger. @mojca opened a ticket here: https://trac.macports.org/ticket/63912

IMO we can revisit this Portfile if aquaterm ever starts working on Tiger.